### ISSUE TITLE
[Bootstrap] Creates .git directory

### ISSUE THREAD

**Issue by marekcirkos** [lerna/lerna#438]
Version: 2.0.0-beta.30

Every time I call `learn bootstrap` it creates `.git` directory. Is this expected behaviour? It is a bit frustrating when working with other repos.

Is it possible to disable git integration completely? `--skip-git` does not help.

Best regards

**Comment by gigabo** [comment]
Hi @marekcirkos good question!  It seems like for `bootstrap` it shouldn't be necessary.

Interested in tracking down what's causing it? 😁

**Comment by marekcirkos** [comment]
I already did :). 
You initialise it `Repository` constructor https://github.com/lerna/lerna/blob/d707386f1327e09a9e8f8530fff470e9fb24838c/src/Repository.js#L11

**Comment by gigabo** [comment]
Oh!  Nice find!

Well... maybe that could be more targeted.  What commands actually _need_ to initialize it? :thinking:

**Comment by marekcirkos** [comment]
Lol, I guess I edited my comment during your post :)

It gets allocated in BaseCommand for every single command https://github.com/lerna/lerna/blob/master/src/Command.js#L18

**Comment by natew** [comment]
Any update here? I'm trying to run lerna from Dockerfile, running into this which would require adding unnecessary git dep to docker.

**Comment by noherczeg** [comment]
@natew I'm looking into this right now.

**Comment by malyw** [comment]
Hey @noherczeg , thanks for revisiting this.
In my case we use Git for development, but then deliver project for the customers, which can use anything.
In that case we want to avoid "lerna botstrap" to throw an error "This is not a git repository" as it still may be fully functional without it.
I tried you solution with version 2.0.0-rc4, but seems still there is a problem:
```bash
> lerna "bootstrap" "--skip-git-check"

lerna info version 2.0.0-rc.4
lerna ERR! ENOGIT This is not a git repository, did you already run `git init` or `lerna init`?
```

**Comment by evocateur** [comment]
This is fixed by #803
